aspy/refactor_imports/classify.py

```diff
@@ -88,9 +89,22 @@ def _get_module_info(module_name, application_dirs):
         try:
             fileobj, filename, _ = imp.find_module(module_name)
         except ImportError:
-            # In the general case we probably can't import the modules because
-            # our environment will be isolated from theirs.
-            pass
+            # Sweep for zipimports on sys.path
+            for path_entry in sys.path:
+                path_entry_ext = os.path.splitext(path_entry)[-1].lower()
```

Comment:
I just noticed that `zipimport.zipimporter` would likely validate that it's a zip file, so we could switch this to "Easier to Ask Forgiveness".  Although, for performance reasons, likely faster to check the string suffix since I'm assuming `zipimport.zipimporter` will do some file I/O before failing.  Let me know if you'd like to remove this to be more pythonic though.